.spice/datasets/assets.yaml

```diff
@@ -0,0 +1,11 @@
+name: spiceai.datasets.assets
+type: append
+access:
+  read:
+    - "*"
+migrations:
+  - name: create_view
+    sql: |
+    CREATE OR REPLACE VIEW spiceai.datasets.assets as (
+      SELECT asset_id, display_name, eth_address, polygon_address FROM ethpostgres.public.assets
```

Comment:
Should we call it `name` instead of `display_name`? What do other exchanges use for this friendly name?

Reply:
Name